cli/src/torii/routing.rs

```diff
@@ -76,7 +77,17 @@ impl VerifiedQuery {
         }
         wsv.validator_view()
             .validate(wsv, &self.payload.account_id, self.payload.query.clone())
-            .map_err(|err| QueryExecutionFailure::Permission(err.to_string()))?;
+            .map_err(|error| {
+                let error = ValidationFail::from(error);
+                if let ValidationFail::InternalError(mes) = &error {
+                    error!(
+                        error = mes,
+                        "Internal error occurred during query validation, \
+                         is Runtime Validator correct?"
+                    )
+                }
+                error
+            })?;
```

Comment:
Producing side effects in a mapping function isn't very nice, although it looks clear

Reply:
Yes, but function that should be ok with side effects is called `inspect` and it's not stabilized yet. So for now I think we have no other choice